webapp/src/actions.ts

```diff
@@ -402,8 +402,14 @@ export const userLeft = (channelID: string, userID: string, sessionID: string) =
             },
         });
 
-        if (ringingEnabled(getState()) && !channelHasCall(getState(), channelID)) {
-            await dispatch(removeIncomingCallNotification(callID));
+        if (numSessionsInCallInChannel(getState(), channelID) === 0) {
```

Comment:
Noting this because it's a big change. This is what we do in mobile and it seems to simplify things.

Reply:
I implicitly thought we would handle this case server-side and send the unique call ID because this does not protect us against a race, as we still use the channel ID to figure out the locally stored call ID. It's not a huge deal, but it's something to note.